README.md

```diff
@@ -3,163 +3,277 @@
 
-## Features
-
-CoreNEURON can transparently handle all spiking network simulations including gap junction coupling with the fixed time step method. The model descriptions written in NMODL need to be thread safe to exploit vector units of modern CPUs and GPUs. The NEURON models must use Random123 random number generator.
-
+* MOD files should be THREADSAFE
+* MOD files must use Random123 random number generator (instead of MCellRan4)
+* POINTER variable needs to be converted to BBCOREPOINTER ([details here](http://bluebrain.github.io/CoreNeuron/index.html))
 
 ## Dependencies
-* [CMake 3.0.12+](https://cmake.org)
-* [MOD2C](http://github.com/BlueBrain/mod2c)
+* [CMake 3.7+](https://cmake.org)
 * [MPI 2.0+](http://mpich.org) [Optional]
-* [PGI OpenACC Compiler >=18.0](https://www.pgroup.com/resources/accel.htm) [Optional, for GPU systems]
-* [CUDA Toolkit >=6.0](https://developer.nvidia.com/cuda-toolkit-60) [Optional, for GPU systems]
+* [PGI OpenACC Compiler >=19.0](https://www.pgroup.com/resources/accel.htm) [Optional, for GPU support]
+* [CUDA Toolkit >=6.0](https://developer.nvidia.com/cuda-toolkit-60) [Optional, for GPU support]
 
+In addition to this, you will need other [NEURON dependencies](https://github.com/neuronsimulator/nrn) like Python, Flex, Bison etc.
 
 ## Installation
 
-This project uses git submodules which must be cloned along with the repository itself:
+CoreNEURON is now integrated into development version of NEURON simulator. If you are a NEURON user, preferred way to install CoreNEURON is to enable extra build options during NEURON installation as follows:
 
-```
-git clone --recursive https://github.com/BlueBrain/CoreNeuron.git
+1. Clone latest version of NEURON:
 
-```
+  ```
+  git clone https://github.com/neuronsimulator/nrn
+  cd nrn
+  ```
 
-Set the appropriate MPI wrappers for the C and C++ compilers, e.g.:
+2. Create a build directory:
 
-```bash
-export CC=mpicc
-export CXX=mpicxx
-```
+  ```
+  mkdir build
+  cd build
+  ```
 
-And build using:
+3. Load modules : Currently CoreNEURON version rely on compiler auto-vectorisation and hence we advise to use Intel/Cray/PGI compiler to get better performance. This constraint will be removed in near future with the integration of [NMODL](https://github.com/BlueBrain/nmodl) project. Load necessary modules on your system, e.g.
+
+	```
+	module load intel intel-mpi python cmake
+	```
+If you are building on Cray system with GNU toolchain, set following environmental variable:
+
+	```bash
+	export CRAYPE_LINK_TYPE=dynamic
+	```
+
+3. Run CMake with the appropriate [options](https://github.com/neuronsimulator/nrn#build-using-cmake) and additionally enable CoreNEURON with `-DNRN_ENABLE_CORENEURON=ON` option:
+
+  ```bash
+  cmake .. \
+   -DNRN_ENABLE_CORENEURON=ON \
+   -DNRN_ENABLE_INTERVIEWS=OFF \
+   -DNRN_ENABLE_RX3D=OFF \
```

Comment:
I'd leave out the RXD and INTERVIEWS option.  Perhaps a prerequisite is to already know how to build NEURON.
Interviews off is mostly for HPC.

Reply:
@nrnhines : interviews and rx3d are on by default and it fails to build if X11/cython doesnt exist. Hence I was thinking to make it off in README for coreneuron.